### ISSUE TITLE
Documentation is missing any information around StringVar like trace_add

### ISSUE THREAD

**Issue by AlfredoDinunzioTR** [TomSchimansky/CustomTkinter#2457]
I could not find any info on the website about how to use StringVars so that I could understand the difference from tk and ctk.

Using Intellisense I was able to determine that string var does have a trace method but it is deprecated and replaced by trace_add.   Trace_add uses 'write' instead of 'w' like trace. 

Was able to work it out, but think some documentation updates are needed

**Comment by AlfredoDinunzioTR** [comment]
Maybe it is my fault for not realizing that we would just use the regular tk documentation for this.

**Comment by dipeshSam** [comment]
Yeah. It should be added to the customtkinter documention also. It's crucial part of the Tkinter.

@TomSchimansky should look into this matter.

**Comment by AlfredoDinunzioTR** [comment]
We also do not have documentation for CTkCanvas.  Do you want me to create a new issue, or should we update this to be broader for updated documentation.  I am OK without it for now so no urgency. Just want to help people in the future.

**Comment by dipeshSam** [comment]
It's appreciable. Do it at your level. Tom may look for it when he come back.

It's quite clearer that most programmers really want to use updated Tkinter. It's an amazing experience as Tkinter provides huge flexibility which is not a case in Browser Based GUIs, but issues take place when documention found insufficient to build large scale applications. We need to work on documention in possible ways.

Regards.

**Comment by teijiIshida** [comment]
We are missing doc for CTkImage too.
